app/browser/api/ledger.js

```diff
@@ -726,70 +686,29 @@ const updatePublishers = (state, publisherKeys) => {
   return state
 }
 
-// TODO rename function
-const excludeP = (publisherKey, callback) => {
-  let doneP
+const doneP = (err, result, callback) => {
+  callback(err, result)
+}
 
-  const done = (err, result) => {
-    doneP = true
-    callback(err, result)
+const getPublisherExclude = (publisherKey, callback) => {
+  if (!publisherInfoData) {
+    return setTimeout(() => module.exports.getPublisherExclude(publisherKey, callback), 10 * ledgerUtil.milliseconds.second)
```

Comment:
what we will get with 10 sec delay?

Reply:
This should probably be ~15 seconds, it's just that `publisherInfoData` may not be yet available due to startup delay